.github/CODEOWNERS

```diff
@@ -1,9 +1,21 @@
 # Require legal approval for all new graphics
-android/app/src/main/res/drawable*/ @organicmaps/legal
-android/app/src/main/res/fonts/ @organicmaps/legal
-android/app/src/main/res/mipmap*/ @organicmaps/legal
-data/*.ttf @organicmaps/legal
-data/resources*/ @organicmaps/legal
-data/search-icons/ @organicmaps/legal
-data/styles/clear/style-*/ @organicmaps/legal
-iphone/Maps/Images.xcassets/ @organicmaps/legal
+android/app/src/main/res/drawable*/ @organicmaps/design
+android/app/src/main/res/fonts/ @organicmaps/design
+android/app/src/main/res/mipmap*/ @organicmaps/design
+data/*.ttf @organicmaps/design
+data/resources*/ @organicmaps/design
+data/search-icons/ @organicmaps/design
+data/styles/clear/style-*/ @organicmaps/design
+iphone/Maps/Images.xcassets/ @organicmaps/design
+android/ @organicmaps/android
+iphone/ @organicmaps/ios
+xcode/ @organicmaps/ios
+.github/workflows @organicmaps/releng
+android/*gradle* @organicmaps/releng
+xcode/fastlane/ @organicmaps/releng
+data/classificator.txt @organicmaps/styles
```

Comment:
classificator.txt and other *.txt files as well as drules*.bin files are outputs of the script;

probably it makes more sense to trigger reviews only by changes to the source files? e.g. *.mapcss

Reply:
Hey, please suggest changes.